plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -22,6 +22,19 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
     @get:Input
     val tagLatest: Property<String> = objects.property(String::class.java)
 
+    // Variables tag and baseDirectory are not initialised when constructor is called
+    private val dockerRunArgs: List<String> by lazy {
```

Comment:
A function that returns this list might be good enough

Reply:
Done.